src/Components/Open/SampleModels.jsx

```diff
@@ -0,0 +1,86 @@
+import React, {ReactElement, useState} from 'react'
+import {Grid, Chip, Typography} from '@mui/material'
+import {handleBeforeUnload} from '../../utils/event'
+import Momentum from '../../assets/icons/Momentum.svg'
+import Eisvogel from '../../assets/icons/Eisvogel.svg'
+import Seestrasse from '../../assets/icons/Seestrasse.svg'
+import Sheenstock from '../../assets/icons/Sheenstock.svg'
+import Placeholder from '../../assets/icons/Placeholder.svg'
+
+
+/**
+ * @property {Function} setIsDialogDisplayed callback
+ * @property {Function} navigate Callback from OpenModelDialog to change page url
+ * @return {ReactElement}
+ */
+function SampleModelFileSelector({navigate, setIsDialogDisplayed}) {
```

Comment:
This should have the same name as file and have export default on same line here.

Reply:
done